tests/karate/src/test/java/be/constants.feature

```diff
@@ -236,9 +229,9 @@ Feature: Constants
     * def getValidRegisteredVotes = call createValidRegisteredVotes
     * def getInvalidRegisteredVotes = call createInvalidRegisteredVotes
 
-    * def getCoinIssuancePubKey = call createIssuancePubKey
-    * def getCreateSignatureForCoinIssuance = call createSignatureForCoinIssuance
-    * def getValidOutputs =   [{ "value": 32, "script": { "type": "P2PKH", "pubkey_hash": "2jmj7l5rSw0yVb-vlWAYkK-YBwk=" } }]
+    * def getCoinIssuancePubKey = call organizerPk
```

Comment:
Are you sure about this change? It seems contrary to the spec referenced in the original test implementation 🤔

Reply:
As discussed, yes